From 8ec4e2078a0a19ed2ab6bacc5bd31d5f6c65fd66 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Mon, 4 May 2015 10:56:59 +0800
Subject: [PATCH] MLK-9618-6 usb: chipidea: otg: show message for b host HNP
 fails

Show 'Device no response' message if A-device fails to connect as a peripheral.

Signed-off-by: Li Jun <b47624@freescale.com>
---
 drivers/usb/chipidea/otg_fsm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/chipidea/otg_fsm.c b/drivers/usb/chipidea/otg_fsm.c
index 5ad4f50..1d563e2 100644
--- a/drivers/usb/chipidea/otg_fsm.c
+++ b/drivers/usb/chipidea/otg_fsm.c
@@ -371,6 +371,7 @@ static int a_aidl_bdis_tmout(struct ci_hdrc *ci)
 static int b_ase0_brst_tmout(struct ci_hdrc *ci)
 {
 	ci->fsm.b_ase0_brst_tmout = 1;
+	dev_warn(ci->dev, "Device No Response\n");
 	return 0;
 }
 
-- 
1.8.0

